types/matchmediaquery/index.d.ts

```diff
@@ -0,0 +1,40 @@
+// Type definitions for matchmediaquery 0.3
+// Project: https://github.com/ncochard/matchmediaquery#readme
+// Definitions by: Nick Whitlock <https://github.com/eezstreet>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type MediaValues = Record<
+    | 'orientation'
+    | 'scan'
+    | 'width'
+    | 'height'
+    | 'device-width'
+    | 'device-height'
+    | 'resolution'
+    | 'aspect-ratio'
+    | 'device-aspect-ratio'
+    | 'grid'
+    | 'color'
+    | 'color-index'
+    | 'monochrome',
+    unknown
+>;
+
+declare class Mql {
+constructor(query: string, values: Partial<MediaValues>, forceStatic: boolean);
+
+    // addListener(listener: Function | null): void;
+
+    // removeListener(listener: Function | null): void;
```

Comment:
these should be either removed (but this would be not recommented)
or implemented properly.
Otherwise the mql is not really useful at runtime, e.g for changing breakpoints or orientation.

Reply:
Yeah. I commented those out because I couldn't determine what the original signature of the methods were, but I can have a look at it tomorrow and get a better idea.